src/sailfishusermanager.cpp

```diff
@@ -379,9 +403,17 @@ int SailfishUserManager::removeUserFiles(uint uid)
         qCWarning(lcSUM) << "Removing user environment directory failed";
 
     // Execute user removal scripts
-    QDir removal(USER_REMOVE_SCRIPT_DIR);
-    for (const QString &entry : removal.entryList(QStringList() << "*.sh",
-                                                  QDir::Files | QDir::Executable, QDir::Name)) {
+    QDir removal(USER_CREATE_SCRIPT_DIR, "*.sh", QDir::NoSort, QDir::Files | QDir::Executable);
```

Comment:
This should be `USER_REMOVE_SCRIPT_DIR`. Also I think want @spiiroin meant with his previous comment in the other PR was that these both code paths could be combined since they need to only differ by the directory used here. So something like `void executeScripts(uint uid, const QString &directory);`

Reply:
fix it